scripts/manage_user_activity.py

```
@@ -0,0 +1,393 @@
+            return True
+        else:
+            print(f"❌ Failed to create activity: {result.stderr}")
+            return False
+    except subprocess.TimeoutExpired:
+        print("❌ Request timed out")
+        return False
+    except Exception as e:
+        print(f"❌ Error creating activity: {e}")
+        return False
+
+def query_user_activities_via_curl(user_id=None, action=None):
```

Ruff (0.12.2)

Unused function argument: `action`

(ARG001)